packages/styled-system/tests/jest/useStyledSystem.test.tsx

```diff
@@ -0,0 +1,445 @@
+// TODO: add tests for unsafe properties
```

Comment:
Some tests are missing here, but i wanted the PR to be up and reviewed before adding those tests